src/any.rs

```diff
@@ -741,7 +741,9 @@ impl Dynamic {
     }
 
     /// Get a copy of a specific type to the `Dynamic`.
-    /// Casting to `Dynamic` just returns a reference to it.
+    /// Casting to `Dynamic` returns a clone of the value in case of NON-shared
+    /// Dynamic. In case of Shared Dynamic returns a clone of the inner data of
+    /// Shared Dynamic.
     /// Returns `None` if the cast fails.
     #[inline(always)]
     pub fn read<T: Variant + Clone>(&self) -> Option<T> {
```

Comment:
If it clones something, then probably the function name should include the word `clone`.  `read` doesn't convey the idea that a copying is involved.

Reply:
@schungx Will rename to `clone_inner_data`

Reply:
@schungx Done.